src/test/features/response/draft/responseDraft.ts

```diff
@@ -104,81 +104,56 @@ describe('ResponseDraft', () => {
     })
   })
 
-  describe('requireHowMuchPaid', () => {
+  describe('isResponsePartiallyRejectedDueTo', () => {
 
     it('should return false when no response type set', () => {
       const draft: ResponseDraft = new ResponseDraft()
       draft.response = undefined
 
-      expect(draft.requireHowMuchPaid()).to.be.eq(false)
+      expect(draft.isResponsePartiallyRejectedDueTo(undefined)).to.be.eq(false)
     })
 
     it('should return false when response is part admission', () => {
       const draft: ResponseDraft = new ResponseDraft()
       draft.response = new Response(ResponseType.OWE_SOME_PAID_NONE)
 
-      expect(draft.requireHowMuchPaid()).to.be.eq(false)
+      expect(draft.isResponsePartiallyRejectedDueTo(undefined)).to.be.eq(false)
     })
 
     it('should return false when response is part admission without subtype selected', () => {
       const draft: ResponseDraft = new ResponseDraft()
       draft.response = new Response(ResponseType.OWE_SOME_PAID_NONE)
       draft.rejectPartOfClaim = new RejectPartOfClaim(undefined)
 
-      expect(draft.requireHowMuchPaid()).to.be.eq(false)
+      expect(draft.isResponsePartiallyRejectedDueTo('foo')).to.be.eq(false)
```

Comment:
this should probably throw an exception as its an unknown type. better to find the error early